src/js/ripple/amount.js

```diff
@@ -176,13 +189,7 @@ Amount.prototype.ratio_human = function(denominator, opts) {
   opts = extend({ }, opts);
 
   var numerator = this;
-
-  if (typeof denominator === 'number' && parseInt(denominator, 10) === denominator) {
-    // Special handling of integer arguments
-    denominator = Amount.from_json(String(denominator) + '.0');
-  } else {
-    denominator = Amount.from_json(denominator);
-  }
+  denominator = Amount.from_json(noIntegers(denominator));
```

Comment:
The original tacking on of `.0` seems ill considered. 

Perhaps an IOU style amount should be created for these cases where `noIntegers` is used, otherwise the number is parsed as a native amount (which seem to be truncated to 6 decimal places (iirc))